docs/resources/waypoint_application.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_waypoint_application Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  Waypoint Application resource
+---
+
+# hcp_waypoint_application (Resource)
+
+Waypoint Application resource
```

Comment:
nit: The application data source description ended with a `.`, so I'm suggesting this here to be consistent 😄 

```suggestion
Waypoint Application resource.
```

Reply:
That's fair. For future reference, this is generated from the schema. I updated the description in e7fc320